user-operator/src/main/java/io/strimzi/operator/user/UserController.java

```diff
@@ -172,69 +169,27 @@ protected void stop() {
                 LOGGER.debugOp("Interrupted while stopping controller loop", e);
             }
         });
+
+        InformerUtils.stopAll(5_000L, userInformer, secretInformer);
```

Comment:
Is it not safer to stop the informers first, and wait for the queues to empty before stopping the controller?

Reply:
Good idea. I added it there. Thanks.